### ISSUE TITLE
Component:  Dropdown input missing aria label

### ISSUE THREAD

**Issue by WTW-Jacob-Evans** [primefaces/primeng#15093]
### Describe the bug

When a p-dropdown is set to editable, the input is missing an aria label. This is causing the issue to be flagged by Axe Dev Tools.
![image](https://github.com/primefaces/primeng/assets/163871134/190d91d4-ce77-4105-83cc-b3010d543abe)
 
![image](https://github.com/primefaces/primeng/assets/163871134/ed021fbb-fbad-456f-810d-30763cd09d67)

These issues all seem to stem from the input box when a dropdown is made editable.

### Environment

windows 10

### Angular version

17.2.3

### PrimeNG version

17.10.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

**Comment by cetincakiroglu** [comment]
Hi,

This issue is duplicate of https://github.com/primefaces/primeng/issues/15122 and it's fixed in https://github.com/primefaces/primeng/pull/15162

Closing the issue, thanks for reporting!
